### PR TITLE
DEV-3386-new-go-exercise-only-b

### PR DESCRIPTION
**Pull request by jotapero**
Created PR

### CODE REVIEW

**Review by tiagocollot — APPROVED** [review]

File: tests/onlyb_test/main.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
```
Comment (tiagocollot) [inline tests/onlyb_test/main.go:2]: I think we should increase the coverage os tests.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: tests/onlyb_test/main.go (hunk 1 above)
Comment (jotapero) [reply tests/onlyb_test/main.go:2]: Same answer as in onlyF exercise

**Review by tiagocollot — APPROVED** [review]
LGMT

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**